misk/web/tabs/README.md

```diff
@@ -111,64 +111,12 @@ Configuring the Misk Service
 
 Adding your Tab Webpack Build to Gradle
 ---
-Tab builds are kicked off by Gradle but done within a Docker container for portability across environments. The following steps will update your Gradle to be able to start the `misk-node` Docker container and to build your tab in it.
+Tab builds are kicked off by Gradle but done within a Docker container for portability across environments.
 
-Add the following code to your service's `dependencies.gradle`.
-
-```JSON
-  "dockerGradlePlugin": "gradle.plugin.com.palantir.gradle.docker:gradle-docker:0.20.1",
-```
-
-If this is an internal Square service, leave the value of the above blank and pull the latest `polyrepo` pinned version with `./polyrepo deb-upgrade {service name}`.
-
-In your service's central `build.gradle` add the following where appropriate. Primarily, you may need to add the maven repository pointing to `https://plugins.gradle.org/m2/` and `classpath dep.dockerGradlePlugin` to your dependencies. If your service has no subprojects, that section can be ignored.
+In your service's project `build.gradle` file you will need to add the following to configure the Docker plugin, start the container, and let Gradle spin off a build if there is a change in your tab code. Adjust the template below to fit your service's file structure and to use the most up to date [Docker image version](https://hub.docker.com/r/squareup/).
```

Comment:
we really want a MiskGradlePlugin so people don’t have to copy so much config into their builds!

Reply:
100!